inventories/nos_event_service/inventory.yml

```diff
@@ -9,6 +9,10 @@ nos_event_service:
     events_google_pubsub_credentials_json_path: '{{inventory_dir}}/group_vars/all/vault_pubsub-credentials.json'
     events_listen_address: 8008
     events_environment: PRODUCTION
+    events_pyroscope_application_name: events.nos.social
```

Comment:
This isn't blocking for this PR or deployment, but I would generally recommend moving this variable down to the hosts specific section of the inventory file, or make it more general, like `events_pyroscope_application_name: "{{ inventory_hostname }}"`.    This set of variables apply to all hosts in the nos_event_service inventory group, and so is too hard-coded if we ever decided to have multiple event services running under different domains.

Reply:
Almost!  https://github.com/planetary-social/ansible-scripts/commit/c66f1efa8c39243c37295bd34b550d4bc577fbd2 sets it at the group level, and so all servers in the prod group would have the same app name, regardless of their domain.  We want to set it at [the host level](https://docs.ansible.com/ansible/latest/inventory_guide/intro_inventory.html#assigning-a-variable-to-one-machine-host-variables).  Again, this is a bit pedantic for this case, as there's only a single server.  It's hopefully helpful/interesting though.

I pushed a change(https://github.com/planetary-social/ansible-scripts/pull/65/commits/5324e03d7e0f536929aaba182c6bb55531c541bc) that sets that variable to specifically our events.nos.social server.

With that push, I think this is good to merge!